applications/plugins/SofaPython/Binding_Base.cpp

```diff
@@ -586,6 +585,67 @@ static PyObject * Base_getListOfLinks(PyObject *self, PyObject * /*args*/) {
     return pyList;
 }
 
+
+/// This function is called by the Python interpreter when calling (dir).
+/// It add to the default behavior all the data fields and links of the object.
```

Comment:
/// It adds to the default behavior all the data fields and links of the object.